src/mockServerLambda.js

```diff
@@ -33,5 +33,5 @@ exports.verifyInvocation = (mockServerClient, functionName, requestBody, times)
       json: JSON.stringify(requestBody),
       matchType: 'ONLY_MATCHING_FIELDS'
     }
-  }, times, true);
+  }, times, times);
```

Comment:
Here is the primary breaking MockServer API change

Reply:
For anyone following along - here's the breaking change: https://github.com/jamesdbloom/mockserver-client-node/commit/dce5b438a845da9bb5353a430bb911c1cd12ecd8#diff-f884a33a5b565b9215e48c815f97e3eeR417 (v5.5.0) Their docs are still outdated, though: https://github.com/jamesdbloom/mockserver-client-node#verify-requests